.travis.yml

```diff
@@ -21,10 +21,12 @@ matrix:
   allow_failures:
     - rust: nightly
   include:
-    - sudo: required
-      os: linux
+    - os: linux
       rust: nightly-2016-04-10
       env: BUILD_ENV=-arm_cross_compile
+    - os: linux
+      rust: nightly
```

Comment:
I wish we could build against the pinned version of nightly. But we can allow failures only on [groups of jobs](https://docs.travis-ci.com/user/customizing-the-build/#Rows-that-are-Allowed-to-Fail).